.appveyor.yml

```diff
@@ -1,30 +0,0 @@
-environment:
-  matrix:
-  - julia_version: 1
-  SCIPOPTDIR: C:\scipoptdir
-
-platform:
-  - x64 # 64-bit
-
-branches:
-  only:
-    - master
-
-notifications:
-  - provider: Email
-    on_build_success: false
-    on_build_failure: false
-    on_build_status_changed: false
-
-install:
-  - ps: iex ((new-object net.webclient).DownloadString("https://raw.githubusercontent.com/JuliaCI/Appveyor.jl/version-1/bin/install.ps1"))
-  - ps: wget https://scip.zib.de/download/release/SCIPOptSuite-7.0.0-win64-VS15.exe -outfile scipopt-installer.exe
```

Comment:
Should we re-add appveyor with the download updated to latest patch?

Reply:
Well, maybe. this change was merged from another branch where I wanted to migrate to GitHub actions.
Re-adding the AppVeyor CI would only make sense to me if we give up on SCIP_jll for Windows, I guess.